pkg/image/testdata/v2/single_manifest/manifests/oc-mirror

```diff
@@ -0,0 +1 @@
+sha256:d31c6ea5c50be93d6eb94d2b508f0208e84a308c011c6454ebf291d48b37df19
```

Comment:
I'm not sure why an image SHA is here. Is this for testing reason?

Reply:
Might need to add this to `.gitignore`. Getting generated by the unit test.